hypha/apply/activity/context_processors.py

```diff
@@ -5,5 +5,5 @@ def notification_context(request):
     context_data = dict()
     if hasattr(request, 'user'):
         if request.user.is_authenticated and request.user.is_apply_staff:
-            context_data['latest_notifications'] = Activity.objects.latest().order_by('-timestamp')[:5]
+            context_data['latest_notifications'] = Activity.objects.filter(current=True).latest().order_by('-timestamp')[:5]
```

Comment:
The `Activity` model class already has `Meta.ordering`, so there is no need to declare `order_by` here again and if the ordering is used there is no need to use `.latest()`.

```suggestion
            context_data['latest_notifications'] = Activity.objects.filter(current=True).all()[:5]
```

or, if we really want to enforce the ordering here, use: 
```
Activity.objects.filter(current=True).all().order_by('-timestamp')[:5]
```

Reply:
It looks like `latest()` is a [custom queryset implementation](https://github.com/HyphaApp/hypha/blob/0288e814da3bf4172c6feb0dee9aa952cab3a537/hypha/apply/activity/models.py#L58-L59), so it makes sense to keep it here. 

That said, why not use an appropriate custom queryset for current activities instead of using `current=True` everywhere?

Reply:
Yeah, that is also a good idea but It is not that widely used right now, it is only used in one or two places. So, I think it is fine for now.